prog/vnet/subnet_nexus.rb

```diff
@@ -172,7 +175,7 @@ def self.random_private_ipv6(location, project)
     network_mask = NetAddr::Mask128.new(64)
     selected_addr = NetAddr::IPv6Net.new(network_address, network_mask)
 
-    selected_addr = random_private_ipv6(location, project) if project.private_subnets_dataset[Sequel[:net6] => selected_addr.to_s, :location => location]
+    selected_addr = random_private_ipv6(location, project) if project.private_subnets_dataset[Sequel[:net6] => selected_addr.to_s, :location_id => location.id]
```

Comment:
Realize the existing code was this way, but there's not a reason to use `Sequel[:net6]` instead of `:net6` here:

```suggestion
    selected_addr = random_private_ipv6(location, project) if project.private_subnets_dataset[net6:  selected_addr.to_s, location_id: location.id]
```

Similar for the net4 case below.